src/renderer/components/DropDownSelector.js

```diff
@@ -65,13 +65,13 @@ const ButtonContainer = styled.div`
 `;
 
 type Props = {
-  children: (props: { isOpen: ?boolean, value: ?DropDownItemType }) => React$Node,
-  items: DropDownItemType[],
+  children: (props: { isOpen: ?boolean, value: ?DropDownItemType<> }) => React$Node,
+  items: DropDownItemType<>[],
```

Comment:
Do we really need <> ? 
Isn't putting nothing put any by default ? (just asking because i find this weird but if it's not possible it's fine)

Reply:
Yes looks like it... I tried but looks like Flowtype is enforcing it for generic types even with a default value 😞